src/core/edgeNetwork/injectSendEdgeNetworkRequest.js

```diff
@@ -94,17 +90,6 @@ export default ({
             response
           })
           .then(returnValues => {
-            // This line's location is very important.
-            // As long as we received a properly structured response,
-            // we consider the response sucessful enough to call lifecycle
-            // onResponse methods. However, a structured response from the
-            // server may ALSO containing errors. Because of this, we make
-            // sure we call lifecycle onResponse methods, then later
-            // process the warnings and errors.
-            // If there are errors in the response body, an error will
-            // be thrown here which should ultimately reject the promise that
-            // was returned to the customer for the command they executed.
-            processWarningsAndErrors(response);
```

Comment:
In case you're looking at this diff and noticing that we were calling `processWarningsAndErrors` twice. Yeah, I should have removed this line with my prior commit. Leaving it there was causing warnings and errors to be logged twice. I didn't catch it previously because I thought they were being logged for separate events during my manual testing. Anyway, what was `validateNetworkResponseIsWellFormed` has now been merged with `processWarningsAndErrors` and it's only being called once up higher.